idxmgmt/manager_test.go

```diff
@@ -481,18 +419,14 @@ func (h *mockClientHandler) Load(config template.TemplateConfig, _ beat.Info, fi
 	return nil
 }
 
-func (h *mockClientHandler) CheckILMEnabled(mode libilm.Mode) (bool, error) {
-	if mode == libilm.ModeDisabled {
+func (h *mockClientHandler) CheckILMEnabled(mode bool) (bool, error) {
+	if !mode {
```

Comment:
maybe rename the variable `mode` since `auto` is gone and it's now just representing enabled/disabled?